### PR TITLE
Write Set Up steps in linear Getting Started guide

### PR DESCRIPTION
**Pull request by rmartz**
# Overview
This writes the set up steps as a linear Getting Started guide, hopefully to make it more intuitive for a user who is unfamiliar with the project and the motivation behind steps to follow.

Goals are:
- Steps should be able to be performed in a linear fashion.
- Each step should be accomplishable by itself, and not require context or instructions from later in the document to understand and complete.
- Avoid cases of "don't know what you don't know"
  - No experience in administering a DRIVER environment should be assumed, and any areas where building a DRIVER-Android application intersects with artifacts assumed to have been created as part of a DRIVER deploy or relies on concepts related to deploying a DRIVER environment should be explicitly called out.
- Avoid ambiguous terminology
  - Paths should be clear in if they are referencing a DRIVER-Android path or DRIVER path
  - Terms should be referred to using consistent language

This should contain all the same information as before, only structured in a more linear fashion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,92 +4,108 @@
...
+```
+
+### (Optional) Set up Single-Sign-On (SSO)
+This will enable users to sign in with their Google account using OpenID, instead of requiring them
+to have a username and password set up.
+
+> **Note** The Android app requires users to have at least Analyst permissions within DRIVER to use, but users created by SSO default to Public permissions. Users created this way will need to have their account permissions upgraded by a administrator user to be able to use the app.
+
+- Create Android OAuth credentials in the [Google APIs Developer Console](https://console.developers.google.com/)
+- For OAuth to work your environment must already be configured with OAuth credentials for front-end us - it may be appropriate to configure these under the same account.
```
Comment (pcaisse) [inline README.md]: I think there's a typo here (`s/us/use`)

File: README.md [hunk 2]
```diff
@@ -4,92 +4,108 @@
...
-See the steps under [Setup](#setup), below. If you need to deploy to an existing stack or set up a new stack
-then you should read this whole document thoroughly.
+## Getting Started
+To set up Android Studio to build and run this project, there are a few steps you need to perform first.
 
-## Generating a Keystore
-A valid keystore is required for most of the remaining steps. If you are developing an app that is
-targeted at an existing installation of DRIVER, then that installation should have a pre-existing
-keystore that you can use.
+- Generate a keystore
```
Comment (pcaisse) [inline README.md]: This is not strictly necessary, but you may want to have these items link to the corresponding section within the document, like a table of contents.

For reference: https://gist.github.com/asabaylus/3071099

**Review by pcaisse — APPROVED** [review]
This is great! It reads much more nicely now. Very clear and thorough. Thanks for taking the time to do this!

Left a couple of small comments.

File: README.md (hunk 1 above)
Comment (rmartz) [reply README.md]: Good catch, took a look over and I think I found a couple other `a/an` errors that I fixed up as well

**Pull request merged**